site/src/components/LicenseBanner/LicenseBanner.test.tsx

```diff
@@ -9,8 +9,8 @@ import { Language } from "./LicenseBannerView"
 describe("LicenseBanner", () => {
   it("does not show when there are no warnings", async () => {
     render(<LicenseBanner />)
-    const bannerPillSingular = await screen.queryByText(Language.licenseIssue)
-    const bannerPillPlural = await screen.queryByText(Language.licenseIssues(2))
+    const bannerPillSingular = screen.queryByText(Language.licenseIssue)
+    const bannerPillPlural = screen.queryByText(Language.licenseIssues(2))
```

Comment:
`screen.queryByText`, `screen.getByRole`, etc. don't return Promises. There's no need to `await` them.

[`@typescript-eslint/await-thenable`](https://typescript-eslint.io/rules/await-thenable)